pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -619,6 +612,27 @@ func (r *ReconcileChaosEngine) reconcileForDelete(request reconcile.Request) (re
 
 }
 
+func (r *ReconcileChaosEngine) removeChaosResources(engine *chaosTypes.EngineInfo, request reconcile.Request) (reconcile.Result, error) {
+	optsDelete := []client.DeleteAllOfOption{
+		client.InNamespace(request.NamespacedName.Namespace),
+		client.MatchingLabels{"engineUID": string(engine.Instance.UID)},
+		client.PropagationPolicy(metav1.DeletePropagationForeground),
+	}
+	if err := r.client.DeleteAllOf(context.TODO(), &appsv1.Deployment{}, optsDelete...); err != nil {
+		return reconcile.Result{}, fmt.Errorf("Unable to delete chaosEngine allocated Chaos Resources, due to error: %v", err)
```

Comment:
Can you please change the msg to mention the resource type like `Unable to delete Deployments allocated to the chaosEngine, due to error: %v", err)` so that it would be easier to detect the failure